packages/retail-ui-extensions/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../config/typescript/tsconfig.admin.json",
```

Comment:
@vividviolet @alanthai why does the admin have its own overrides for this? They seem unnecessary since the TS outputs are not actually for consumers, they are purely used as types

Reply:
@lemonmade I ended up changing this to extend `"extends": "../../config/typescript/tsconfig.base.json"`

Reply:
@lemonmade good catch! This is legacy and should be deleted. We used to use the typescript compiler to build our bundles that's why we had our own settings. @alanthai do you have time to delete this stuff since you're ATC this week?